backend/src/test/resources/application-default.yaml

```diff
@@ -26,6 +26,56 @@ simple-report:
     role-prefix: "SR-UNITTEST-TENANT:"
   feature-flags:
     patient-links: true
+  site-admin-emails:
+    - ruby@example.com
+  demo-users:
+    alternate-users:
```

Comment:
i imagine this is a question you labored over a bunch already, but why is Bob not a `default-user` here?

Reply:
well since i'm absolutely the most likely person to mess that up and i already know, i think it's less urgent, but sure, couldn't hurt to add a comment.

Reply:
I added an assertNull in BaseApiTest instead (and a comment!). Gets more attention than just a comment.